nuxt.config.js

```diff
@@ -49,7 +49,11 @@ export default {
    ** Plugins to load before mounting the App
    ** https://nuxtjs.org/guide/plugins
    */
-  plugins: ['@/plugins/vue-scrollactive', '@/plugins/sanity-client', {src: '@/plugins/fullstory', mode: 'client'}],
+  plugins: [
+    '@/plugins/vue-scrollactive',
+    '@/plugins/sanity-client',
```

Comment:
Are we removing `sanity` from the codebase entirely at this point?

Reply:
Due to pricing and functionality, we'll standardize on Contentful for the public website and therefore docs. Though I did prefer many things about Sanity.

Reply:
Awesome. I’m great with Contentful! I think just removing the plugin ref should be good then.